Apps/Sandcastle/gallery/iTwin Demo.html

```diff
@@ -0,0 +1,194 @@
+          2199023255694: "Building Wall",
+          2199023255696: "Building Window",
+        };
+
+        let selectedFeature;
+        let picking = false;
+
+        Sandcastle.addToggleButton(
+          "Per-feature selection",
+          false,
+          function (checked) {
+            picking = checked;
+            if (!picking) {
+              unselectFeature(selectedFeature);
+            }
+          },
+          "checkbox",
+        );
+
+        // Set up viewer
+        const viewer = new Cesium.Viewer("cesiumContainer", {
+          terrain: Cesium.Terrain.fromWorldTerrain(),
+        });
+        const scene = viewer.scene;
+        scene.globe.show = true;
+        scene.debugShowFramesPerSecond = true;
+
+        // HTML overlay for showing feature name on mouseover
+        const nameOverlay = document.createElement("div");
+        viewer.container.appendChild(nameOverlay);
+        nameOverlay.className = "backdrop";
+        nameOverlay.style.display = "none";
+        nameOverlay.style.position = "absolute";
+        nameOverlay.style.bottom = "0";
+        nameOverlay.style.left = "0";
+        nameOverlay.style["pointer-events"] = "none";
+        nameOverlay.style.padding = "4px";
+        nameOverlay.style.backgroundColor = "black";
+        nameOverlay.style.whiteSpace = "pre-line";
+        nameOverlay.style.fontSize = "12px";
+
+        function selectFeature(feature, movement) {
+          feature.color = Cesium.Color.clone(
+            Cesium.Color.fromCssColorString("#eeff41"),
+            feature.color,
+          );
+          selectedFeature = feature;
+
+          nameOverlay.style.display = "block";
+          nameOverlay.style.bottom = `${
+            viewer.canvas.clientHeight - movement.endPosition.y
+          }px`;
+          nameOverlay.style.left = `${movement.endPosition.x}px`;
+          const element = feature.getProperty("element");
+          const subcategory = feature.getProperty("subcategory");
+          const message = `
+      Element ID: ${element}
+      Subcategory: ${classes[subcategory] || subcategory}
+      Feature ID: ${feature.featureId}`;
+          nameOverlay.textContent = message;
+        }
+
+        function unselectFeature(feature) {
+          if (!Cesium.defined(feature)) {
+            return;
+          }
+
+          feature.color = Cesium.Color.clone(Cesium.Color.WHITE, feature.color);
+          selectedFeature = undefined;
+          nameOverlay.style.display = "none";
+        }
+
+        const statusOutput = document.querySelector("#status");
+        async function init() {
+          statusOutput.innerText = "Starting export";
+
+          const start = Date.now();
+          const delay = (ms) => new Promise((res) => setTimeout(res, ms));
+
+          statusOutput.innerText = "Creating Tileset";
+          let tileset;
+          try {
+            tileset = await Cesium.ITwinData.createTilesetFromModelId(imodelId);
+          } catch (error) {
```

Comment:
Do we expect this case to happen really ever for the Sandcastle sample data?

If not, I think for simplicity's sake it should be removed.

I do think it would be helpful to be moved to a tutorial or similar, as discussed in https://github.com/CesiumGS/cesium/pull/12289#discussion_r1850829211.